modules/installation/pages/install-proxy.adoc

```diff
@@ -11,11 +11,20 @@ include::_partials/entities.adoc[]
 endif::[]
 
 
-This chapter explains how to install and set up {productname} {productnumber} Proxy.
-It also provides notes about migrating a previous proxy to version 3.2.
 
-[[at.manager.proxy.concepts]]
-== Overview
+{susemgr} Proxy is a {suse} base product from version 4.0.
+This section describes how to install {susemgr} Proxy from {sls} installation media.
+It assumes you have already registered the {susemgr} product with the {scc} and have a registration code.
+
+For information on registering with {scc}, or obtaining installation media, see xref:[installation-general-requirements]
+// I can't remember how we're supposed to do xrefs. Need to check with @jcayouette --LKB
+
+[NOTE]
+====
+If you are installing {susemgr} as a proof of concept, or this is a test installation, you can follow these instructions on a virtual machine rather than in production.
```

Comment:
This sentence isn't Simple English ;)  And then I'm wondering why you do not want to install the server on a VM even in production if the VM is powerful enough.

Reply:
Good catch. Updated.